# This message provides essential diagnostics information for Crocoddyl solvers.

time stamp

uint16 iterations
float32 total_time
float32 solve_time
float32 cost
float32 regularization
float32 step_length
float32 dynamic_feasibility
float32 equality_infeasibility
float32 inequality_infeasibility